cmd/kubelet/app/options/options.go

```diff
@@ -230,20 +227,27 @@ func getLabelNamespace(key string) string {
 	return ""
 }
 
-// NewKubeletConfiguration will create a new KubeletConfiguration with default values
+// NewKubeletConfiguration will create a new internal KubeletConfiguration with default values.
 func NewKubeletConfiguration() (*kubeletconfig.KubeletConfiguration, error) {
-	scheme, _, err := kubeletscheme.NewSchemeAndCodecs()
+	versioned, err := utilcodec.NewKubeletConfigurationE()
```

Comment:
TODO(mtaufen): not sure if it makes sense to have a value constructor in the codec helper package.